libraries/chain/contracts/evt_contract.cpp

```diff
@@ -709,12 +709,9 @@ apply_evt_approvedelay(apply_context& context) {
 
         delay_def delay;
         tokendb.read_delay(adact.name, delay);
-        EVT_ASSERT(delay.status == delay_status::proposed, delay_status_exception, "Delay is not in 'proposed' status.");
+        EVT_ASSERT(delay.status == delay_status::proposed, delay_status_exception, "Delay is not in proper status.");
```

Comment:
It's true that delay is not in `proposed` status. Here 'proper' is ok but 'proposed' provides more detail.